src/fragments/lib/push-notifications/common/remote-media.mdx

```diff
@@ -0,0 +1,5 @@
+Amazon Pinpoint supports adding remote media URLs to push notifications.
```

Comment:
This sentence seems unnecessary to me with the follow up on it from iOS - isn't it? on its own it doesn't make much sense I feel.

Reply:
removed